app/src/main/java/ch/sdp/vibester/Register.kt

```diff
@@ -0,0 +1,119 @@
+package ch.sdp.vibester
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import android.widget.TextView
+import android.widget.Toast
+import ch.sdp.vibester.auth.FireBaseAuthenticator
+import com.google.android.gms.auth.api.signin.GoogleSignIn
+import com.google.android.gms.auth.api.signin.GoogleSignInClient
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions
+import com.google.android.gms.tasks.Task
+import com.google.firebase.auth.AuthResult
+
+class Register : AppCompatActivity() {
+
+//    private lateinit var googleSignInClient: GoogleSignInClient
+
+    private lateinit var authenticator: FireBaseAuthenticator
+
+    private lateinit var email: TextView
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_google_log_in)
+
+//        val gso = GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
```

Comment:
Consider adding @FIXME or explain in the comment why stop using google sign in to remind us later in case